tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -16,6 +16,7 @@
 #include <ROOT/RNTuple.hxx>
 
 #include <ROOT/RFieldVisitor.hxx>
+#include <ROOT/RNTupleAnchor.hxx>
```

Comment:
Given that the definition of the relevant member functions was moved to RNTupleAnchor.cxx, I would say this is not required here anymore.
```suggestion
```

Reply:
I think we still need it for `RNTupleReader::Open(RNTuple *ntuple, ...)`

Reply:
Oh, that's right :+1:!